### ISSUE TITLE
Map card suggests I name a map that ive already named

### ISSUE THREAD

**Issue by Bortseb** [metamaps/metamaps#881]
I'm not exactly sure how to replicate this one... but I would say that its because the `new` part of the URL never got deleted.... is there any way to tell if that URL is internally self-generated by our code... vs pasted in by someone who named their map after copying/sharing the URL

**Comment by devvmh** [comment]
No, that's not possible. But we could use Backbone  in all cases to check if the map is still "Untitled Map". If we set up backbone to do this, probably we wouldn't even need the ?new in the first place

Replication steps:

1. Create a new map (ensure that the url ends in ?new)
2. Name the map
3. Hit "Refresh" on your browser

**Comment by devvmh** [comment]
can you look into the backbone router code and figure this one out? it's probably not far removed from the issue where you set the title of the page to the map name
